src/main/resources/application-metrics-rhel.yaml

```diff
@@ -0,0 +1,33 @@
+METERING_RHEL_TASK_TOPIC: ${clowder.kafka.topics[?(@.requestedName == 'platform.rhsm-subscriptions.metering-rhel-tasks')].name:platform.rhsm-subscriptions.metering-rhel-tasks}
+SERVICE_INSTANCE_INGRESS_TOPIC: ${clowder.kafka.topics[?(@.requestedName == 'platform.rhsm-subscriptions.service-instance-ingress')].name:platform.rhsm-subscriptions.service-instance-ingress}
```

Comment:
For my reference:
METERING_RHEL_TASK_TOPIC is for incoming 
SERVICE_INSTANCE_INGRESS_TOPIC is for outgoing events